APE13.rst

```diff
@@ -128,9 +131,9 @@ domain-specific classes are not defined in this APE; instead, the intent is for
 `Spectrum1D` to provide a starting point for such efforts, which will then
 develop as they are called for by the community.
 
-The development outlined in this APE will consist of three packages:
+The development outlined in this APE will consist of two packages:
 
-* `specutils <https://github.com/astropy/specutils)>`_.  This will provide the
+* `specutils <https://github.com/astropy/specutils>`_.  This will provide the
   basic interface classes (including the `Spectrum1D` object), as well as
```

Comment:
```suggestion
  basic interface classes (including the `Spectrum1D` object, to be renamed `Spectrum` in specutils 2.0), as well as
```
It might be useful to make a note of this here while you're making these other changes.